### PR TITLE
Feature/hyperparameter scans

### PR DESCRIPTION
**Pull request by nbarlowATI**
* Add a new script `hyperparameter_scan.py` that runs sets of simulations for many sets of hyperparameters and stores the loss function values in a text file.

### CODE REVIEW

**Review by jack89roberts — APPROVED** [review]
LGTM
